cmd/flags_run_local_instance.go

```diff
@@ -83,44 +82,27 @@ func (flags *RunLocalInstanceCommandFlags) MergeToConfig(c *types.Config) (err e
 		c.RunConfig.BridgeName = flags.BridgeName
 	}
 
-	c.RunConfig.Verbose = flags.Verbose
-	c.RunConfig.Bridged = flags.Bridged
-	c.RunConfig.Accel = flags.Accel
-	c.Force = flags.Force
-
-	if flags.IPAddress != "" && isIPAddressValid(flags.IPAddress) {
+	if flags.IPAddress != "" {
```

Comment:
Probably nanos discards invalid IP addresses, but I don't see a reason why not to verify whether the IP address is valid in ops side. It would be good to show some warning messages in the cases that IP address, network mask and default gateway are invalid, so user can fix it.

Reply:
nanos validates the IP address.

Reply:
As I said, even having nanos validating IPs I don't see a reason why not having ops helping users identifying invalid IPs.

Reply:
This is a repetition and it only affects ops-run, while nanos implementation affects all commands consistently.